### ISSUE TITLE
Teleportation to Stormwind Portal Room results in facing wrong direction

### ISSUE THREAD

**Issue by TQPS** [TrinityCore/TrinityCore#26799]
### Description

(Only tested with a Mage) When teleporting to Stormwind, you always arrive in the portal room facing the wrong direction.

### Expected behaviour

You should be facing the opposite portal exit which is 180 degree's from it's current value.

(The current facing portal is not an exit)

### Steps to reproduce the problem

1. Have a mage with Stormwind Teleport
2. Teleport to Stormwind which arrives in the Portal Room
3. You will be facing the wrong direction (a non exit)

### Branch

3.3.5

### TC rev. hash/commit

TrinityCore rev. 87a4bc3 2021-07-23 00:12:58 +0300 (3.3.5 branch) (Win64, RelWithDebInfo, Static)

### Operating system

Windows 7 x64

### Custom changes

Trickerer's Bots

**Comment by offl** [comment]
Are we talking about https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=17334 (updated from sniffs in 2020)
Or about areatrigger in Stormwind? (updated from world safe locs in 2020)
Or about personal teleport https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=3561 ?

In sniff 3561 uses same position as 17334

```sql
--
UPDATE `spell_target_position` SET `PositionX` = -8998.14, `PositionY` = 861.254, `Orientation` = 2.25 WHERE `ID` = 3561;
```

**Comment by TQPS** [comment]
I have only tested the personal one to date. I have not tested the portal version or similar such as Shat to Stormwind.

**Comment by offl** [comment]
Forgot to tell, that commit breaks more orientation values than fixes
That will be fixed with #25273
